systers_portal/meetup/migrations/0006_auto_20160622_1717.py

```diff
@@ -1,18 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import models, migrations
-
-
-class Migration(migrations.Migration):
```

Comment:
Why are the migration files deleted? There seem to be no changes in the model file in this pr.

Reply:
@sa7i As I had to switch and pull after merge conflicts, needed to migrate again- thats why these deleted files are reflected here.